lib/common/resources/otel-collector-config.yml

```diff
@@ -9,7 +9,14 @@ metadata:
   namespace: "{{namespace}}"
 spec:
   mode: "{{deploymentMode}}"
-  image: public.ecr.aws/aws-observability/aws-otel-collector:v0.21.0
+  image: public.ecr.aws/aws-observability/aws-otel-collector:v0.33.1
```

Comment:
Since this is going to impact other OSS patterns, can you run EKS and Graviton EKS OSS pattern and share output.?

Reply:
Tested both OSS and Graviton patterns. Both work as intended with all dashboards working